src/AppBundle/Resources/translations/messages.en.yml

```diff
@@ -1,19 +1,20 @@
 login:
-    title: Login with Tiqr
+    title: Log in with tiqr
     timeout: Login timeout. Please refresh this page and try again.
     push:
-        message: You will receive a push notification on your phone.
+        message: Open the tiqr pushmessage on your phone and follow the instructions.
```

Comment:
er moet nog een spatie tussen "push" en "message", maar ik denk dat het duidelijker is om het "push notification" the noemen.

Reply:
Thanks for the feedback @wesselvanderlinden, I've notified the PO's with a request to review the translations.